src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.Basic.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_Basic
+    {
+        private static readonly byte[] ExtraArrayUtf8 = Encoding.UTF8.GetBytes("ExtraArray");
+        private static readonly byte[] FirstUtf8 = Encoding.UTF8.GetBytes("first");
+        private static readonly byte[] LastUtf8 = Encoding.UTF8.GetBytes("last");
+        private static readonly byte[] AgeUtf8 = Encoding.UTF8.GetBytes("age");
+        private static readonly byte[] PhoneNumbersUtf8 = Encoding.UTF8.GetBytes("phoneNumbers");
+        private static readonly byte[] AddressUtf8 = Encoding.UTF8.GetBytes("address");
+        private static readonly byte[] StreetUtf8 = Encoding.UTF8.GetBytes("street");
+        private static readonly byte[] CityUtf8 = Encoding.UTF8.GetBytes("city");
+        private static readonly byte[] ZipUtf8 = Encoding.UTF8.GetBytes("zip");
+
+        private ArrayBufferWriter<byte> _arrayBufferWriter;
+
+        private int[] _numberArrayValues;
+
+        [Params(true, false)]
+        public bool Formatted;
+
+        [Params(true, false)]
+        public bool SkipValidation;
+
+        [Params(10, 100_000)]
+        public int DataSize;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _arrayBufferWriter = new ArrayBufferWriter<byte>();
+
+            var random = new Random(42);
```

Comment:
Sounds good. I was actually wishing something like this would exist.

It would be great if we could have similar methods for non-ascii text as well:
https://github.com/dotnet/performance/blob/eeae67639c76434596551ca2285e2d540527cd76/src/harness/BenchmarkDotNet.Extensions/ValuesGenerator.cs#L98

Something like this:
https://github.com/dotnet/corefxlab/blob/a908f8cf7cfa4f2a5efa3bdeca36438306d23dbe/tests/System.Text.Primitives.Tests/Encoding/TextEncoderTestHelper.cs#L9-L41